src/sortStudents.ts

```diff
@@ -1,16 +1,74 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const copy = [...students];
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+  const average = (numbers: number[]): number => {
+    return numbers.reduce((a, b) => a + b, 0) / numbers.length;
+  };
```

Comment:
Replace helping function outside of main function
```suggestion
  const getAverage = (numbers: number[]): number => {
    return numbers.reduce((a, b) => a + b, 0) / numbers.length;
  };
```